components/more-less/more-less.js

```diff
@@ -51,11 +51,14 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 	static get styles() {
 		return css`
 			:host {
-				display: block;
+				display: flow-root;
```

Comment:
This corrects the bounding box measurements that are thrown off by the padding/negative-margin trick.